packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -442,6 +442,29 @@ aar artifact already exists, continuing...''',
     createArtifactProgress.complete();
   }
 
+  /// Uploads a release ipa to the Shorebird server.
+  Future<void> createIosReleaseArtifact({
+    required int releaseId,
+    required String ipaPath,
+  }) async {
+    final createArtifactProgress = logger.progress('Creating artifacts');
+    final ipaFile = File(ipaPath);
+    try {
+      await codePushClient.createReleaseArtifact(
+        releaseId: releaseId,
+        artifactPath: ipaPath,
+        arch: 'aarch64',
```

Comment:
```suggestion
        arch: 'ipa',
```

Is this guaranteed to be true for all ipas?

Reply:
Ah good catch. Changed to ipa to be in line with aabs and aars